sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -195,7 +195,11 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
         }
 
         should.equal(error.statusCode, 404, "Error must not be undefined");
-        should.equal(error.code, "404", `Code expected to be "404" but received ${error.code}`);
+        should.equal(
+          error.code,
+          "MessageEntityNotFoundError",
+          `Code expected to be "MessageEntityNotFoundError" but received ${error.code}`
```

Comment:
When the check fails, mocha will print out both the expected and actual values. Therefore, neither of them need to be repeated in the message

```suggestion
          `Unexpected error code found.`
```

Reply:
Same for a few other places in this file.

Reply:
Updated